notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1549 @@
+{
```

Comment:
<p>fix grammar -&gt; "In order to run training in Vertex, we define a python file that includes all the code from preprocessing to training. Most of the code is the same as what we have in the cells above. But we will add a few additional lines of code to do hyperparameter training. These lines will save the trained mode and the preprocessing states for use later.</p><p> </p><p>We will use the <code>hypertune</code> library to report the <code>accuracy</code> as the metric for optimization.  "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>